.github/release_schedule.json

```diff
@@ -0,0 +1,8 @@
+{
+    "2022-05-26": "0.15.7",
+    "2022-06-02": "0.15.8",
+    "2022-06-09": "0.15.9",
+    "2022-06-16": "0.15.10",
+    "2022-06-23": "0.15.11",
+    "2022-06-30": "0.15.12"
+}
```

Comment:
This JSON configuration adheres to a particular schema based on a test in `test_release.py`. It must be a set of key-value pairs with datetimes corresponding to semvers. The schedule must be non-empty, kept up-to-date, increase versions sequentially, etc. Any deviations will cause the test to fail and the build to exit early.